### DISCUSSION TITLE
What does OpenSSH do in gsconnect?

### DISCUSSION THREAD

**Discussion by fxzxmicah** [andyholmes/gnome-shell-extension-gsconnect#1789] [Q&A] [answered]
When I installed gsconnect, i saw fedora listing openssh as a dependency. So I want to ask what openssh does in gsconnect? After all, I can't intuitively feel which function needs openssh to run.

**Comment by daniellandau** [comment, marked as answer]
Mounting a folder uses it.

**Reply by fxzxmicah** [reply]
Mount a folder from another computer to local computer through ssh protocol?

**Reply by daniellandau** [reply]
Yes
